webpack.config.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path')
+const webpack = require('webpack')
+const SimpleProgressWebpackPlugin = require('simple-progress-webpack-plugin')
+// const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+// const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+module.exports = {
+  target: 'web',
+  entry: {
+    backgroundPage: './add-on/src/background/background.js',
+    browserAction: './add-on/src/popup/browser-action/index.js',
+    pageAction: './add-on/src/popup/page-action/index.js',
+    uploadPage: './add-on/src/popup/quick-upload.js',
+    optionsPage: './add-on/src/options/options.js',
+    proxyAclManagerPage: './add-on/src/pages/proxy-acl/index.js',
+    proxyAclDialog: './add-on/src/pages/proxy-access-dialog/index.js'
+  },
+  optimization: {
+    /*
+    occurrenceOrder: true,
+    minimizer: [
+      new UglifyJsPlugin({
```

Comment:
Any reason why we can't enable uglify here?

Reply:
When we run webpack with `-p` it is enabled by default with default options. 
I've left this section in case we want to customize options that are passed to uglify, but defaults seem to works just fine. Will probably remove it before merge.